doc/version_history.rst

```diff
@@ -2,6 +2,11 @@
 Version History
 ===============
 
+v0.8.1
+-------
+
+* Fix the test related to **ts_m2com** v0.11.1.
```

Comment:
I think this should read v0.12.0, because, as I mentioned in the other PR, that change should be a minor release, not a patch.